core/templates/dev/head/pages/about/about.html

```diff
@@ -42,7 +42,7 @@
             <div class="oppia-static-card-content oppia-static-card-content-narrow">
               <h2>About Oppia</h2>
               <div class="pull-right">
-                <em>Oppia "O-pee-yah" (Finnish) - "to learn"</em>
+                <em style="text-align:center;">Oppia "O-pee-yah" (Finnish) - "to learn"</em>
```

Comment:
Pull this out into a CSS class that is defined at the bottom of this file. Something like oppia-about-tab-subtitle perhaps.